src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -3608,7 +3608,10 @@ namespace ZoneTempPredictorCorrector {
                 if (TempCtrlFound && TempControlledZone(RelativeZoneNum).DeltaTCutSet > 0.0) {
                     TempZoneThermostatSetPoint(ZoneNum) = TempControlledZone(RelativeZoneNum).ZoneThermostatSetPointLo;
                     ZoneThermostatSetPointLo(ZoneNum) = TempControlledZone(RelativeZoneNum).ZoneThermostatSetPointLo;
-                    if (ZoneAirSolutionAlgo == Use3rdOrder) Tprev = ZTM1(ZoneNum);
+                    if (ZoneAirSolutionAlgo == Use3rdOrder) {
+                        Tprev = MAT(ZoneNum);
+                        if (ShortenTimeStepSys) Tprev = XMPT(ZoneNum);
+                    }
                     if (ZoneAirSolutionAlgo == UseAnalyticalSolution || ZoneAirSolutionAlgo == UseEulerMethod) Tprev = ZoneT1(ZoneNum);
```

Comment:
This should be an elseif or simply else. Same comment in other similar blocks.

Reply:
@mjwitte I like it and am going to make changes.